rasa/core/channels/console.py

```diff
@@ -98,6 +98,11 @@ def _get_user_input(previous_response: None) -> Text:
     ...
 
 
+@overload
+def _get_user_input(previous_response: Dict[str, Any]) -> Optional[Text]:
```

Comment:
Why do we need to overload this method?

Reply:
it's because there was another overload at line 97, and mypy complained about incomplete overloads